docs/developers_guide/FlinkComponents.md

```diff
@@ -1,22 +1,72 @@
 # Flink components
```

Comment:
What about section Prerequisites or Concepts?  You could put this link with lifecycle stuff there, but may be there is something conceptually preceding the lifecycle stuff. A link to the starting page for Flink programming model or something like that would help. Do you need to interact only with Nu's overlay on Flink's API or is it a mix of Nu's stuff and Flink?